models/dex/dex_prices_latest_legacy.sql

```diff
@@ -1,7 +1,7 @@
 {{ config(
-        alias = alias('prices_latest'),
+        alias = alias('prices_latest', legacy_model=True),
         tags = ['legacy'],
-        partition_by = ['block_time'],
+        partition_by = ['block_month'],
```

Comment:
we're looking to universally change partitions to monthly level, as our partitions are too small
fyi, partitioning on `block_time` is much too granular, at minimum should be daily (if massive dataset)

Reply:
ok good to know, taking note